modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-enlargement.adoc
+
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
```

Comment:
Suggestion to take or leave: I would personally prefer "expanding" vs. "enlarging" (to match the usage in the linked docs).